docs/docs/Starter-Projects/starter-projects-sequential-agent.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Sequential tasks agent
+sidebar_position: 4
+slug: /starter-projects-sequential-agent
+---
+
+Build a **Sequential Tasks Agent** flow for a multi-agent application using multiple **Agent** components.
+
+Each agent has an LLM model and a unique set of tools at its disposal, with **Prompt** components connected to the **Agent Instructions** fields to control the agent's behavior. For example, the **Researcher Agent** has a **Tavily AI Search** component connected as a tool. The **Prompt** instructs the agent how to answer your query, format the response, and pass the query and research results on to the next agent in the flow.
+
+Each successive agent in the flow builds on the work of the previous agent, creating a chain of reasoning for solving complex problems.
+
+## Prerequisites
+- [OpenAI API key created](https://platform.openai.com/)
+- [Tavily AI API key created](https://www.tavily.com/)
+
+## Open Langflow and create a new flow
+
+1. Click **New Flow**, and then select **Sequential Tasks Agent**.
+This opens a starter template with the necessary components to run the flow.
+
+![Starter flow for Sequential Tasks Agent](/img/starter-flow-sequential-agent.png)
+
+The Sequential Tasks Agent flow consists of these components:
+
+* The **Agent** components use the connected LLM to reason through the user's input and select among the connected tools to complete its task.
+* The **Chat Input** component accepts user input to the chat.
+* The **Prompt** component combines the user input with a user-defined prompt.
+* The **Chat Output** component prints the flow's output to the chat.
+* The **YFinance** tool component provides access to financial data from Yahoo Finance.
+* The **Tavily AI Search** tool component performs AI-powered web searches.
+* The **Calculator** tool component performs mathematical calculations.
+
+## Run the Sequential Tasks Agent flow
+
+1. Add your OpenAI API key to the **Agent** components.
```

Comment:
Is "components" correct or should it be "component"?

Reply:
There are 3 in the application so it's "components"